es-vue-base/src/lib-mixins/forms.js

```diff
@@ -66,6 +66,11 @@ export default {
     },
     methods: {
         validateState(name) {
+            // Check that given field has key in validators object
+            const fieldHasValidators = this.$v.form && this.$v.form[name];
+            if (!fieldHasValidators) {
+                return null;
+            }
```

Comment:
@ericdouglaspratt pointed out the reliance of the `form` namespace in this method. This might be an opportunity to break that reliance.

Reply:
Is the suggestion to make the mixin functional without the existence of a `form` key in the `data` function?

Reply:
That's right. The usage of `validateState(fieldName)` would now become `validateState(form.fieldName)` which is more "correct" imo.